src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceApiServiceSpec.scala

```diff
@@ -439,7 +450,17 @@ class WorkspaceApiServiceSpec extends BaseServiceSpec with WorkspaceApiService w
           }
         }
       }
+
+    // how to add queries?
+    "Passthrough tests on the workspaces/tags path" - {
+      "OK status is returned for GET" in {
+        stubRawlsService(HttpMethods.GET, tagAutocompletePath, OK)
+        Get(Uri(tagAutocompletePath).withQuery(("q", "tag"))) ~> dummyUserIdHeaders("1234") ~> sealRoute(workspaceRoutes) ~> check {
```

Comment:
A note on this test. You're doing two things here, stubbing out a rawls call to `tagAutocompletePath` and then making an API call to Orchestration on the same path. This works if and only if orchestration and rawls share the same endpoint path. That isn't always the case and I've seen bugs where the two are assumed to be the same, but aren't. 

I think I would prefer that you use the orchestration path url explicitly here to avoid that confusion. You did that above in the `MethodNotAllowed` test.